app/static/css/index.css

```diff
@@ -0,0 +1 @@
+/* Some styling */
```

Comment:
is there though??

Reply:
a lil, if you use the world of imagination :(